timelapse.py

```diff
@@ -62,8 +63,19 @@ def layer_changed(timelapse_folder, webcam_url, webcam_http_auth, webcam_https_v
     else:
         log_print('Failed to get timelapse snapshot.', file=sys.stderr)
 
+def create_video(timelapse_path, current_log_print, snapshots_path, keep_snapshots):
+    video_file = os.path.abspath(os.path.join(timelapse_path, current_log_print + ".mp4"))
+    snapshots_files = snapshots_path + os.path.sep + "*.jpg"
+    cmd = "ffmpeg -r 20 -y -pattern_type glob -i '" + snapshots_files + "' -vcodec libx264 " + video_file
```

Comment:
I sometimes need different ffmpeg options, frame rates, etc. depending on my print job and model.
How about we make this whole command string an CLI argument so every user can specify their own set of options? This would also revert the preferred behaviour of *not* running ffmpeg by default, only if the user specifies a e.g., `--ffmpeg-command="ffmpeg -r 20 -y <...>"` argument.

Reply:
I couldn't find a clever solution to pass all arguments in a clean way, so decided to leave it as is by default. 
There is always the option to disable the default command and execute your own command.
Maybe a config file for the video command? I don~t have much time to expand this feature.